webpack/scenes/ContentViews/ContentViewsConstants.js

```diff
@@ -10,6 +10,8 @@ export const PUBLISH_CONTENT_VIEW_KEY = 'CONTENT_VIEW_PUBLISH';
 export const UPDATE_CONTENT_VIEW = 'UPDATE_CONTENT_VIEW';
 export const UPDATE_CONTENT_VIEW_SUCCESS = 'UPDATE_CONTENT_VIEW_SUCCESS';
 export const UPDATE_CONTENT_VIEW_FAILURE = 'UPDATE_CONTENT_VIEW_FAILURE';
+export const ADD_CONTENT_VIEW_PACKAGE_FILTER_KEY = 'ADD_CONTENT_VIEW_PACKAGE_FILTER_KEY';
+export const DELETE_CONTENT_VIEW_PACKAGE_FILTER_KEY = 'DELETE_CONTENT_VIEW_PACKAGE_FILTER_KEY';
 export const cvDetailsKey = cvId => `${CONTENT_VIEWS_KEY}_${cvId}`;
```

Comment:
ADD_CONTENT_VIEW_PACKAGE_GROUP_FILTER_RULE or if that seems too long ADD_CV_PACKAGE_GROUP_FILTER_RULE.. I like the 2nd one better..

Reply:
:+1: